src/capwap_http_api_handler.erl

```diff
@@ -342,10 +360,10 @@ fmt_ip(IP) ->
     erlang:list_to_binary(Res).
 
 fmt_wtp_descriptor(#wtp_descriptor{
-			 max_radios = MaxRadios,
-			 radios_in_use = RadiosInUse,
-			 encryption_sub_element = EncSubElem,
-			 sub_elements = SubElements}) ->
+                      max_radios = MaxRadios,
```

Comment:
white space cleanups are ok, but they should go into a extra commit that touches only white space